sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -112,8 +116,9 @@ protected void append(@NotNull ILoggingEvent eventObject) {
         CollectionUtils.filterMapEntries(
             loggingEvent.getMDCPropertyMap(), entry -> entry.getValue() != null);
     if (!mdcProperties.isEmpty()) {
-      if (!options.getContextTags().isEmpty()) {
-        for (final String contextTag : options.getContextTags()) {
+      final List<String> contextTags = HubAdapter.getInstance().getOptions().getContextTags();
```

Comment:
This is a bit confusing, we have the `options` property, but we read from `HubAdapter.getInstance().getOptions()`.
Should we rather get rid of the `options` then?

Reply:
Yes, that's why i created #2053 I just didn't want to do it now as this quick fix solves the current problem and further improvements didn't seem urgent